src/main/java/am/itspace/sweetbakerystore/controller/web/UserAccountController.java

```diff
@@ -37,4 +43,5 @@ public String userAddressPage() {
     public String userEditAddress() {
         return "redirect: web/user-account/edit-address";
     }
+
```

Comment:
I have feelings url 
redirect: web/user-account/edit-address";
will not working.